codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -680,6 +681,7 @@ public ChannelFuture goAway(final ChannelHandlerContext ctx, final int lastStrea
             // Need to retain before we write the buffer because if we do it after the refCnt could already be 0 and
             // result in an IllegalRefCountException.
             debugData.retain();
+            release = true;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "release". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)